json-logs/samples/api/chat.postMessage.json

```diff
@@ -89,7 +89,9 @@
                   }
                 ],
                 "style": "",
-                "indent": 12345
+                "indent": 12345,
+                "offset": 12345,
```

Comment:
These JSON files are auto-updated when you run the e2e tests under test_with_remote_apis in slack-api-client sub project. Also, these files are the sources of type definitions in the Node SDK package: https://github.com/slackapi/node-slack-sdk/tree/main/packages/web-api/src/response